SR2020/2020-06-25-virtual-competition-dates-and-robot-celebrations.md

```diff
@@ -0,0 +1,46 @@
+---
+to: sr2020-teams
+subject: Competition Dates & Celebrating Your Robots
+---
+
+Hi,
+
+### Dates
+
+We're pleased to announce the dates for the virtual competition. As previously
+announced the competition will run over three weekends in July.
+
+Teams will be able to submit their code in advance and the simulated matches
+will be broadcast live via a live stream over the weekend.
+
+Weekend         | Matches
+----------------|---------
+11th July 2020  | League
+18th July 2020  | League
+25th July 2020  | Knockout
+
+We'll be publishing more details of the competition, including how code
+submission will work and exactly when live streams will be live nearer the time.
+
+
+### Celebrating your robots
+
+We would also like to celebrate the progress made towards your physical robots
+prior to the transition to a virtual competition. We appreciate that many teams
+will not have had access to their robots during the lock-down, so will not be
+awarding prizes for this, however we would still like to recognise the effort
+which the teams had put in.
+
+We would like to hear from you if your team has any designs, photos or video of
+your physical robots which you would like to share for inclusion in the live
+stream broadcast.
+
+We are also interested in recording remote interviews with team members
```

Comment:
Is there a CTA for this bit?

Reply:
Apparently not. Should we just ask them to email teams@?

Reply:
Yeah emailing works.

Also, The requirement of media consent will mean updating our template.

Reply:
Hadn't thought about template update. Pretty sure we'd need the consent though?

Reply:
anyway: 4e559cc2c9c05465332536aae70723952013e148